README.md

```diff
@@ -0,0 +1,261 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> List of technologies used for this project
+
+<details>
+  <summary>BackEnd</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org">Ruby on Rails</a></li>
+    <li><a href="https://stimulus.hotwired.dev/">Stimulus</a></li>
+    <li><a href="https://rspec.info/">Rspec</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Front-end/View</summary>
+  <ul>
+    <li><a href="https://react.dev/">React</a></li>
+    <li><a href="https://axios-http.com/es/docs/intro">Axios</a></li>
+    <li><a href="https://tailwindcss.com/docs/guides/ruby-on-rails">Tailwind CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Front-end/Store management</summary>
+  <ul>
+    <li><a href="https://react-redux.js.org/">React-redux</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Create and Remove Cars**
+- **Create Reservation**
+- **Full stack webpage**
+- **Can see all the list of reservation of current user**
+- **Details of every cars in db**
+- **Log in and sign up feature**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo Link](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+you need to install ruby 3.2.2 using the prefer tool, example: 
+- [rvm](https://rvm.io/)
+- [rbenv](https://github.com/rbenv/rbenv)
+- [asdf-vm](https://asdf-vm.com/)
+
+- Browser (Google Chrome, Mozilla Firefox, Safari or any other browser)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder
+  git clone https://github.com/enmanuelbayen/futuristic-transportation.git
+```
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd futuristic-transportation
+  bundle install
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+./bin/dev
+```
+**Note:** If there are any errors consider adding your necesary username and password in `database.yml` for setting up your local enviroment and use the following comand:
+
+```sh
+rails db:drop db:create db:migrate
+./bin/dev
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+rspec
+```
```

Comment:
- [ ] When I try to run the tests according to instructions, this happens:

![image](https://github.com/Luiscarlosvd/futuristic-transportation/assets/57421823/54d3e41a-1d37-44b6-95c0-65c9e56949fc)

This is because your `requests` tests are `rswag` tests.

## Solution

Just be more specific on your readme:

```
rspec ./spec/models
```

Result:

![image](https://github.com/Luiscarlosvd/futuristic-transportation/assets/57421823/4fc2d59f-3354-405c-977c-88596f960b70)

That way, all your tests will pass. 👍